cfg/param_update.yaml

```diff
@@ -0,0 +1,11 @@
+# File types
+file_common: "_common.cpp"
+file_ros: "_ros.cpp"
+file_cfg: ".cfg"
+file_yaml: ".yaml"
+
+# Path to yaml folder
+path_yaml_folder: "/home/srd-rd/catkin_ws/src/ipa_navigation/ipa_navigation_config/config/components"
+
+# Default editor to open files in (command line name of executable)
+editor: "subl"    # sublime
```

Comment:
we could use `gedit`, `vi/vim` allows opening multiple files, though it is not very convenient.

Fallback to `$EDITOR` would be great.